docs/getting-started.md

```diff
@@ -88,6 +89,50 @@ We can use the redis image based on alpine linux from the DockerHub.
 The containerd client uses the `Opts` pattern for many of the method calls.
 We use the `containerd.WithPullUnpack` so that we not only fetch and download the content into containerd's content store but also unpack it into a snapshotter for use as a root filesystem.
 
+Let's put the code together and see it in action.
+
+```go
+package main
+
+import (
+        "context"
+        "log"
+
+        "github.com/containerd/containerd"
+        "github.com/containerd/containerd/namespaces"
+)
+
+func main() {
+        if err := redisExample(); err != nil {
+                log.Fatal(err)
+        }
+}
+
+func redisExample() error {
+        client, err := containerd.New("/run/containerd/containerd.sock")
+        if err != nil {
+                return err
+        }
+        defer client.Close()
+
+        ctx := namespaces.WithNamespace(context.Background(), "example")
+        image, err := client.Pull(ctx, "docker.io/library/redis:alpine", containerd.WithPullUnpack)
+        if err != nil {
+                return err
+        }
+        log.Printf("Successfully pulled %s image\n", image.Name())
+
+        return nil
+}
+```
+
+```bash
+> go build -o example main.go
```

Comment:
Too many balls up in the air this morn, I should have caught the 1.7 and 1.8 error there.  But this document doesn't mention that need that I can find and it would be good to mention that.  I think your list above looks great, I'd say go with it along with your suggestion for the "Go version 1.8x" and call it soup.

Thanks again for working this through.  In case you haven't guessed one of my pet peeves are examples that break if you do only what they say, ie. they're missing other often times basic setup steps.